packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -437,6 +467,61 @@ const hideVideoLoadingSpinner = async (page: Page): Promise<void> => {
   await page.addStyleTag({ content: '[data-ui-id="stream-media-loading-spinner"] {display: none}' });
 };
 
+const MASK_ATTRIB_KEY = 'data-ui-id';
+const MASK_ATTRIB_VALUE = 'stream-media-video-mask';
+const MASK_SELECTOR = `[${MASK_ATTRIB_KEY}="${MASK_ATTRIB_VALUE}"]`;
+/**
+ * Masks the video element in {@link VideoTile} by overlaying an opaque <div/>.
```

Comment:
Can you add a comment here why someone would want to mask the videos in their test/why we are masking videos? (Just hey wow artifacts killed our UI test stability so applying a mask here!)